tests/test_fake_toas.py

```diff
@@ -236,6 +236,60 @@ def test_fake_fromMJDs(MJDs):
     assert np.isclose(r.calc_time_resids().std(), 1 * u.us, rtol=0.2)
 
 
+def test_fake_fromMJDs_keepmean():
+    # basic model, no EFAC or EQUAD
+    model = get_model(
+        io.StringIO(
+            """
+        PSRJ J1234+5678
+        ELAT 0
+        ELONG 0
+        DM 10
+        F0 1
+        F1 -1E-15
+        PEPOCH 58000
+        """
+        )
+    )
+    t1 = np.linspace(57001, 57500, 100, dtype=np.longdouble) * u.d
+    t2 = np.linspace(57501, 58000, 100, dtype=np.longdouble) * u.d
+    toas1 = pint.simulation.make_fake_toas_fromMJDs(
+        t1,
+        model=model,
+        error=1 * u.us,
+        add_noise=True,
+    )
+    toas2 = pint.simulation.make_fake_toas_fromMJDs(
+        t2,
+        model=model,
+        error=1 * u.us,
+        add_noise=True,
+    )
```

Comment:
This test enforces that the default for `subtract_mean` is True. Is that desired?